src/images/svg/developer-programme.svg

```diff
@@ -0,0 +1,23 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="23" height="24" viewBox="0 0 24 23">
+    <g fill="none" fill-rule="evenodd">
+        <g>
```

Comment:
Could you please minify this by running `svgo src/images/svg/developer-programme.svg`? 🙏

Reply:
Done